packages/embeds/embed-react/inline.tsx

```diff
@@ -20,6 +20,19 @@ function App() {
         action: "*",
         callback,
       });
+
+      api("ui", {
+        cssVarsPerTheme: {
+          light: {
+            "cal-border-booker": "red",
+            "cal-border-booker-width": "20px",
+          },
+          dark: {
+            "cal-border-booker": "red",
```

Comment:
Do we default these to red ? Or was this just for testing?

Reply:
This is actually a test page for inline embed. 
Not clear from the name though. Will rename it in a followup

Reply:
Gotcha :) Other than that LGTM